docs/docs/upgrade/network-api-migration.md

```diff
@@ -268,7 +274,7 @@ The following table summarizes the API transformations when using the network AP
     "container": {
         “type”: “DOCKER”,
         "docker": {
-            “image”: “foo”,
+            “image”: “image-name”,
```

Comment:
the `image` field in the second column doesn't match. it should. either go with `foo` or `image-name` for both